packages/patternfly-4/react-table/src/components/Table/utils/decorators/collapsible.tsx

```diff
@@ -34,22 +37,23 @@ export const collapsible = (value, { rowIndex, columnIndex, rowData, column, pro
   };
 };
 
-export const expandable = (value, { rowData }) =>
+export const expandable = (value: IFormatterValueType, { rowData }: IExtra) =>
   rowData.hasOwnProperty('parent') ? <ExpandableRowContent>{value}</ExpandableRowContent> : value;
 
-export const expandedRow = colSpan => {
+export const expandedRow = (colSpan: number) => {
   const expandedRowFormatter = (
-    value,
+    value: IFormatterValueType,
     {
       rowIndex,
       rowData,
       column: {
         extraParams: { contentId = 'expanded-content' }
       }
-    }
+    }: IExtra
   ) =>
     rowData.hasOwnProperty('parent') && {
-      colSpan: colSpan + !!rowData.fullWidth,
+      // todo: rewrite this logic, it is not type safe
```

Comment:
We should probably reference this somewhere so we don't forget about it, maybe in #2673 ?

Reply:
yes, this is also not really Typescript compatible... will add to #2673